extensions/amazon-lambda-resteasy/runtime/src/test/java/io/quarkus/amazon/lambda/resteasy/adapter/test/ResteasyAwsProxyTest.java

```diff
@@ -1,408 +0,0 @@
-package io.quarkus.amazon.lambda.resteasy.adapter.test;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
-
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.StringJoiner;
-import java.util.UUID;
-
-import javax.ws.rs.core.HttpHeaders;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
-
-import org.apache.commons.codec.binary.Base64;
-import org.jboss.resteasy.plugins.server.servlet.ResteasyContextParameters;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
-
-import com.amazonaws.serverless.proxy.internal.LambdaContainerHandler;
-import com.amazonaws.serverless.proxy.internal.servlet.AwsServletContext;
-import com.amazonaws.serverless.proxy.internal.testutils.AwsProxyRequestBuilder;
-import com.amazonaws.serverless.proxy.internal.testutils.MockLambdaContext;
-import com.amazonaws.serverless.proxy.model.AwsProxyRequest;
-import com.amazonaws.serverless.proxy.model.AwsProxyResponse;
-import com.amazonaws.services.lambda.runtime.Context;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-
-import io.quarkus.amazon.lambda.resteasy.adapter.test.model.MapResponseModel;
-import io.quarkus.amazon.lambda.resteasy.adapter.test.model.SingleValueModel;
-import io.quarkus.amazon.lambda.resteasy.adapter.test.provider.CustomExceptionMapper;
-import io.quarkus.amazon.lambda.resteasy.adapter.test.provider.ServletRequestFilter;
-import io.quarkus.amazon.lambda.resteasy.runtime.container.ResteasyLambdaContainerHandler;
-
-/**
- * Unit test class for the RESTEasy AWS_PROXY default implementation
- */
-public class ResteasyAwsProxyTest {
```

Comment:
These tests should not be deleted, in my draft I adapted them to the virtual channel. These tests picked up quite a few different potential problems that can be caused by using the API incorrectly.

Reply:
these tests as written on my branch don't work with the virtual channel bits.  we can copy your updated tests over, though.